From e900a102e111197f4ff379d6a764742ab98e9adb Mon Sep 17 00:00:00 2001
From: Olivier Masse <olivier.masse@nxp.com>
Date: Mon, 21 Jan 2019 11:45:53 +0100
Subject: [PATCH 19/21] MMIOT-30 MA-13967-1 Add secure_ion.h

secure_ion.h added in drivers includes to expose it in user space

Change-Id: I3adeafc18323e751fccf954bf48e42f3ae34a40c
Signed-off-by: Olivier Masse <olivier.masse@nxp.com>
---
 drivers/staging/android/uapi/secure_ion.h | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)
 create mode 100644 drivers/staging/android/uapi/secure_ion.h

diff --git a/drivers/staging/android/uapi/secure_ion.h b/drivers/staging/android/uapi/secure_ion.h
new file mode 100644
index 0000000..0dfb9d7
--- /dev/null
+++ b/drivers/staging/android/uapi/secure_ion.h
@@ -0,0 +1,25 @@
+/*
+ * drivers/staging/android/uapi/secure_ion.h
+ *
+ * Copyright (C) 2019 NXP, Inc.
+ *
+ * This software is licensed under the terms of the GNU General Public
+ * License version 2, as published by the Free Software Foundation, and
+ * may be copied, distributed, and modified under those terms.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
+
+#ifndef _LINUX_SECURE_ION_H
+#define _LINUX_SECURE_ION_H
+#define DWL_ION_ENCODED_BUFFER_OPTEE_HEAP_MASK         4
+#define DWL_ION_ENCODED_BUFFER_OPTEE_HEAP_NAME         "optee"
+#define DWL_ION_DECODED_BUFFER_DISPLAY_HEAP_MASK       2
+#define DWL_ION_DECODED_BUFFER_DISPLAY_HEAP_NAME       "display"
+#endif
+
-- 
2.7.4

